cmd/cmd.go

```diff
@@ -29,15 +30,20 @@ import (
 
 	"github.com/jmorganca/ollama/api"
 	"github.com/jmorganca/ollama/format"
+	"github.com/jmorganca/ollama/parser"
 	"github.com/jmorganca/ollama/progressbar"
 	"github.com/jmorganca/ollama/readline"
 	"github.com/jmorganca/ollama/server"
 	"github.com/jmorganca/ollama/version"
 )
 
 func CreateHandler(cmd *cobra.Command, args []string) error {
-	filename, _ := cmd.Flags().GetString("file")
-	filename, err := filepath.Abs(filename)
+	filename, err := cmd.Flags().GetString("file")
```

Comment:
This looks like an undocumented change in behavior? Previously it would use a relative `Modelfile` if one existed

Reply:
That doesn't change. The file is can still be relative to where the user calls `ollama create`. It only resolves it to an absolute path